pkg/kubernetes/errors.go

```diff
@@ -0,0 +1,9 @@
+package kubernetes
+
+import "github.com/pkg/errors"
+
+var (
+	errSyncingCaches     = errors.New("Failed initial cache sync for Namespace informers")
+	errInitInformers     = errors.New("Namespace informers are not initialized")
```

Comment:
In light of this informer not being specific to `Namespaces`, and because additional context can be a part of the log message:
```suggestion
	errInitInformers     = errors.New("Informer not initialized")
```